services/api/src/roles.json

```diff
@@ -3,14 +3,16 @@
     "name": "Super Admin",
     "allowScopes": ["global", "organization"],
     "permissions": {
-      "users": "read-write"
+      "users": "read-write",
+      "audit-entries": "read"
```

Comment:
This should be `auditEntries` - that's the convention we use in all the other projects